scrapers/covid-19/getVaccine.js

```diff
@@ -1,9 +1,13 @@
 const axios = require('axios');
+const axiosCookieJarSupport = require('axios-cookiejar-support').default;
+const tough = require('tough-cookie');
 const csv = require('csvtojson');
 const cheerio = require('cheerio');
 const logger = require('../../utils/logger');
 const { months, phaseData } = require('../../utils/RAPS');
 
+axiosCookieJarSupport(axios);
+
 const cleanData = (data) => {
```

Comment:
why dont we add this to /utils/RAPS

Reply:
If the fix works I'll do it at some point later in the week, a lot of code reuse between vaccine/therepeutics you are right

Reply:
made this an issue #949 doing it later time